ipn/ipnlocal/peerapi_test.go

```diff
@@ -455,6 +456,7 @@ func TestFileDeleteRace(t *testing.T) {
 		},
 		rootDir: dir,
 	}
+	ps.b.capFileSharing = hasCapability(ps.b.netMap, tailcfg.CapabilityFileSharing)
```

Comment:
at this point there's no need for lines 451-454.

and also, this line can just be "capFileSharing: true" up above.

Reply:
This way `hasCapability` function won't be covered by the tests, but OK, will change.